validation-test/Sema/type_checker_crashers_fixed/rdar27017206.swift

```diff
@@ -1,4 +1,4 @@
-// RUN: not --crash %target-swift-frontend %s -parse
+// RUN: %target-swift-frontend %s -parse
 // REQUIRES: asserts
```

Comment:
Drive-by comment: please don't forget to remove this from fixed compiler-crashers test cases.